MotorCtrl[] motor
uint32 motor_mask    # 要控制哪几个电机
---
bool succeed
